### DISCUSSION TITLE
Sets Relation complement using bar

### DISCUSSION THREAD

**Discussion by hamaney** [typst/typst#4370] [Q&A] [answered]
I am trying to do the bar over the relationship letter 

<img width="21" alt="image" src="https://github.com/typst/typst/assets/11434409/cc90f202-a9e7-45cc-8ff5-fe33165214d4">

**Comment by hamaney** [comment, marked as answer]
This is what worked for me. 

#let relDash = $attach(R,t: -,tr: "")$

But not sure why "tr" has to be passed to show dash at the top?. if "tr" is not passed then the "t:" makes the dash shows up at the top right.

**Reply by MDLC01** [reply]
It would be more semantically accurate to use the [`overline`](https://typst.app/docs/reference/math/underover#functions-overline) function.

The reason why simply passing a value for `t` does not work is explained in [the documentation for `attach` `t` argument](https://typst.app/docs/reference/math/attach#functions-attach-parameters-t). Instead of passing a value for `tr` to force `t` to attach as limits (i.e., above the content it attaches to), you can use the [`limits`](https://typst.app/docs/reference/math/attach#functions-limits) function.

**Reply by hamaney** [reply]
Thanks the limit function simplifies things
